src/lib/esnext.arraybuffer.d.ts

```diff
@@ -0,0 +1,58 @@
+interface ArrayBuffer {
+	/**
+	 * Read-only. The maximum length that this ArrayBuffer can be resized to (in bytes).
```

Comment:
The spec proposal points out that this is equal to `byteLength` if options.maxByteLength is not provided. I'd stay closer to the example's explanation:

```suggestion
	 * If this ArrayBuffer is resizable, returns the maximum byte length given during construction; returns the byte length if not.
```